pages/api/webhooks/order-created.ts

```diff
@@ -23,14 +24,25 @@ const handler: NextApiHandler = async (request, response) => {
     return;
   }
 
-  const body: OrderCreatedEventSubscriptionFragment = request.body;
+  const saleorDomain = request.headers[SALEOR_DOMAIN_HEADER];
 
-  // FIXME: this part of settings will be fetched from App.metadata and defined based
-  // on channnel used in order.
-  const taxJarConfig = getTaxJarConfig();
+  const body: OrderCreatedEventSubscriptionFragment = request.body;
 
   if (body?.__typename === "OrderCreated") {
     const order = body.order!;
+
+    const taxJarConfig = await getTaxJarConfig(
+      saleorDomain as string,
```

Comment:
Is it always set? What if it was not?

Reply:
As above, it is checked by app-sdk

Reply:
That means `app-sdk` is not very typescript friendly now, is it?

Reply:
@mmiszy The newer version of app-sdk  gives me a better typing. I don't need to mark it `as string` there. The https://github.com/saleor/saleor-taxjar/pull/6/files introduces a beter usage of app-sdk. I will drop `as string` there